src/main/java/Duke.java

```diff
@@ -39,7 +39,7 @@ public static void main(String[] args) {
                         break;
                     case "done":
```

Comment:
Maybe we should align switch statement and Case statement as suggested by Java Coding Standard?